src/utils/propertyTreeHelpers.js

```diff
@@ -157,3 +144,51 @@ export const isDeadEnd = (propertyOwners, properties, uri) => {
   });
   return nonDeadEndSubowners.length === 0;
 }
+
+// Returns true if the URI has the correct format of a scene graph node.
+// OBS! Does not actually check if the SGN exists
+export function isSceneGraphNode(uri) {
+  if (!uri) return false;
+
+  const splitUri = uri.split('.');
+  // A scene graph node URI har the format 'Scene.<NodeIdentifier>'
+  return (splitUri.length == 2);
```

Comment:
dont know that it matters in this context but thinks like RenderEngeine.* will have length 2 and not be a sgn

Reply:
That's true. Just copied this function over from the PropertyOwner file, but should change it so that it explicitly checks for the `Scene` key as well, to be more general